src/test/java/fm/last/moji/integration/AbstractMojiIT.java

```diff
@@ -133,4 +133,62 @@ String newKey() {
     return newKey(RandomStringUtils.randomAlphanumeric(16));
   }
 
+  private void initMogileTestData() throws IOException {
+    if(initMogileTestDataIsDone) {
```

Comment:
Yeap but it can only use in static method.  Another option is update junit to version 5 since it has @BeforeAll to solve the problem, though it is unstable currently.